### DISCUSSION TITLE
Inter-chromosomal peak-to-peak and peak-to-gene prediction

### DISCUSSION THREAD

**Discussion by yanw3** [GreenleafLab/ArchR#2030] [Questions / Documentation]
I would like to check the interactions, including peak-to-peak and peak-to-gene interctions, between chromosomes. But it seems that addCoAccessibility and addPeak2GeneLinks, as well as other current tools do not support such prediction using scATAC data. They even rarely discuss this in the tutorial or corresponding manuscript. I was wondering if anybody can provide suggestions to achieve this. Thank you!

**Comment by rcorces** [comment]
This is not recommended. The reason that you dont see this done is because it explodes the number of comparisons required.

**Reply by yanw3** [reply]
Thank you for your reply!

The inter-chromosomal interaction is an important part to more comprehensively understand the gene expression regulation and understand the relationships between chromatin structure and genome function.

While some techniques, such as Hi-C, can partially achieve the goal of exploring chromatin 3D structure, they may not be accessible to most laboratories. If we can analyze inter-chromosomal interactions using the extensive ATAC data, it would be beneficial for many researchers.

When we have the chance to have the inter-chromosomal results, people will figure out the explosion of the number of increased comparisons in their ways.

Do you have any suggestions for accomplishing this or could you please provide the source code for addCoAccessibility and addPeak2GeneLinks?

Thanks again for your help!
